vulcanize.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Script to vulcanize all the Polymer files into one big blob.
```

Comment:
I take it that this script is not hooked into our build process?  Is there a way to do so?